src/silo/query_engine/actions/nuc_mutations.cpp

```diff
@@ -1,37 +1,41 @@
-#include "silo/query_engine/actions/nuc_mutations.h"
 
 #include <cmath>
 #include <map>
+#include <utility>
+#include <vector>
 
 #include <tbb/blocked_range.h>
 #include <tbb/parallel_for.h>
 #include <nlohmann/json.hpp>
 
+#include "silo/query_engine/actions/nuc_mutations.h"
+
 #include "silo/common/nucleotide_symbols.h"
 #include "silo/database.h"
 #include "silo/query_engine/operator_result.h"
 #include "silo/query_engine/query_parse_exception.h"
 #include "silo/query_engine/query_result.h"
 #include "silo/storage/database_partition.h"
+#include "silo/storage/sequence_store.h"
 
 using silo::query_engine::OperatorResult;
 
 namespace {
 
-std::pair<std::vector<size_t>, std::vector<size_t>> splitBitmaps(
-   const silo::Database& database,
+std::pair<std::vector<size_t>, std::vector<size_t>> preFilterBitmaps(
+   const silo::SequenceStore& sequence_stores,
```

Comment:
This should be singular
```suggestion
   const silo::SequenceStore& sequence_store,
```